day_7/10_speckled_frogs.js

```diff
@@ -0,0 +1,12 @@
+
+var numberOfFrogs = 10
+
+var numberWords = ['zero','one','two','three','four',
+'five','six','seven','eight','nine', 'ten']
+
+for (var i = numberOfFrogs; i > 0; --i) {
+  console.log(`${numberWords[i]} speckled frogs sat on a log
```

Comment:
I believe the rhyme changes based on the number of frogs left. How would you account for that?